src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig

```diff
@@ -34,21 +34,34 @@
   {% block javascrips %}{% endblock %}
   {% block stylesheets %}{% endblock %}
 </head>
-<body class="lang-{{ ps.isoUser }}{% if ps.isRtlLanguage %} lang-rtl{% endif %} ps_back-office bootstrap">
+
+{# Construct CSS classes based on version numbers #}
+{% set major_version = constant('PrestaShop\\PrestaShop\\Core\\Version::MAJOR_VERSION') %}
+{% set minor_version = constant('PrestaShop\\PrestaShop\\Core\\Version::MINOR_VERSION') %}
+{% set release_version = constant('PrestaShop\\PrestaShop\\Core\\Version::RELEASE_VERSION') %}
+{% set body_major = 'body-v' ~ major_version %}
+{% set body_minor = body_major ~ '-' ~ minor_version %}
+{% set body_patch = body_minor ~ '-' ~ release_version %}
+
+<body class="lang-{{ ps.isoUser }}{% if ps.isRtlLanguage %} lang-rtl{% endif %} ps_back-office bootstrap {{ body_major }} {{ body_minor }} {{ body_patch }}">
   <div id="login">
     <div id="content">
       <div id="login-panel">
         <div id="login-header">
-          <h1 class="text-center">
-            <img id="logo" src="{{ ps.baseImgUrl }}prestashop@2x.png" width="123px" height="24px" alt="PrestaShop" />
+          <h1 class="text-center mb-0">
+            <img id="logo" src="{{ ps.baseImgUrl }}prestashop@2x.png" width="128" height="auto" alt="PrestaShop" loading="lazy" />
           </h1>
+          <div class="login__version text-center">
```

Comment:
Login page should not include ps_version.

Reply:
Ok, I have removed this part ✅